test/test_aio.py

```diff
@@ -492,3 +492,24 @@ async def test_transport_closed():
     # No send after transport closed
     with pytest.raises(asyncio_dgram.TransportClosed):
         await stream.send(b"junk", ("127.0.0.1", 0))
+
+
+
+@pytest.mark.asyncio
+async def test_bind_reuse_port():
+    async def use_socket(addr, reuse_port=None):
+        sock = await asyncio_dgram.bind(addr, reuse_port=reuse_port)
+        # give gather time to move to the other uses after the bind
+        await asyncio.sleep(0.1)
+        sock.close()
+
+    addr = ("127.0.0.1", 53001)
+    clients_count = 10
+    with pytest.raises(OSError):
```

Comment:
I'd add a `match` here as there's a lot of potential things that could raise OSError.

`with pytest.raises(OSError, match="Address already in use")`